package.json

```
@@ -45,9 +45,11 @@
     "eslint": "7.x",
     "eslint-plugin-babel": "5.x",
     "eslint-plugin-enact": "0.x",
+    "eslint-plugin-jest": "^24.0.0",
```

I don't know why but let's follow the version rule.
```suggestion
    "eslint-plugin-jest": "24.x",
```